src/main/java/seedu/address/logic/commands/DeleteRecurringSessionCommand.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TIME;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.session.RecurringSession;
+import seedu.address.model.session.Session;
+import seedu.address.model.session.SessionDate;
+import seedu.address.model.student.Name;
+import seedu.address.model.student.Student;
+
+/**
+ * Deletes a session from a recurring session identified using it's displayed index from the address book, date, and
+ * time of the session to be deleted. This command splits a recurring session into two recurring sessions into an
+ * earlier recurring session (if any) and a later recurring session (if any).
+ */
+public class DeleteRecurringSessionCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete_rec_session";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes a session in a recurring session identified by an index number used in the student's"
+            + " list of sessions and the date of the session\n"
+            + "Parameters: n/FULLNAME i/INDEX (must be a positive integer belonging to a recurring session)"
+            + " d/DATE t/TIME (must be a valid date and time of the recurring session)\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "John Doe " + PREFIX_INDEX + "1 "
+            + PREFIX_DATE + "2021-03-30 " + PREFIX_TIME + "12:00";
```

Comment:
Not too sure if this is mentioned before, but is time really needed? since the recurring sessions should have the same time for all.
(doesn't affect functionality, but looking at a user experience pov)

Reply:
Yes not needed, but was trying to use the abstraction of a `SessionDate` to feed through. Will try having a dummy `Time` for 1.4.